metadata-ingestion/src/datahub/emitter/mcp.py

```diff
@@ -104,16 +130,75 @@ def validate(self) -> bool:
             return False
         return True
 
-    def to_obj(self, tuples: bool = False) -> dict:
-        return self.make_mcp().to_obj(tuples=tuples)
+    def to_obj(self, tuples: bool = False, for_file: bool = False) -> dict:
+        obj = self.make_mcp().to_obj(tuples=tuples)
+        if for_file:
```

Comment:
What does "for_file" mean here?
Why is the use case included as a parameter? Would this be better as an extensible enum called "SerializationMode"

Reply:
Or DeserializationMode etc